libs/shared-atomic-design-components/src/lib/atoms/SocialMediaIcons.tsx

```diff
@@ -4,39 +4,56 @@ import './MediaIcons.scss'
 
 const socialMedia = [
   {
-    icon: 'linkedin',
-    link: 'https://www.linkedin.com/school/redi-school-of-digital-integration/',
+    icon: 'fb',
+    link: 'https://www.facebook.com/redischool/',
+    MUC: 'https://www.facebook.com/redimunich/',
+    NRW: 'https://www.facebook.com/redischoolnrw/'
   },
   {
-    icon: 'twitter',
-    link: 'https://twitter.com/redischool?lang=en',
+    icon: 'meetup',
+    link: 'https://www.meetup.com/en-AU/ReDI-school/',
+    MUC: 'https://www.meetup.com/Tech-Talks-Hosted-by-ReDI-School-Munich/',
+    NRW: 'https://www.meetup.com/ReDI-School-NRW/'
   },
   {
     icon: 'instagram',
     link: 'https://www.instagram.com/redischoolberlin/',
+    MUC: 'https://www.instagram.com/redimunich/',
+    NRW: ''
   },
   {
-    icon: 'fb',
-    link: 'https://www.facebook.com/redischool/',
+    icon: 'linkedin',
+    link: 'https://www.linkedin.com/school/redi-school-of-digital-integration/',
   },
   {
-    icon: 'meetup',
-    link: 'https://www.meetup.com/en-AU/ReDI-school/',
-  },
+    icon: 'twitter',
+    link: 'https://twitter.com/redischool?lang=en',
+  }
 ]
 
-const SocialMediaIcons = () => (
-  <ul className="media-icons__list">
-    {socialMedia.map((item: any) => {
-      return (
-        <li key={item.icon}>
-          <a href={item.link} target="_blank" rel="noopener noreferrer">
-            <Icon icon={item.icon} className="media-icons__icon" />
-          </a>
-        </li>
-      )
-    })}
-  </ul>
-)
+interface Props {
+  name?: string
+}
+
+const SocialMediaIcons = ({name}: Props) => {
+  console.log(`name`, name)
+  const icons = name ? socialMedia.slice(0, 3) : socialMedia;
+  return (
+    <ul className="media-icons__list">
+      {icons.map((item: any) => {
```

Comment:
No need to use `any` here, TS should be able to infer the type dynamically. Otherwise, create an interface to describe the shape of the object.

Reply:
I added an interface to solve this